docs/domains/Domain.md

```diff
@@ -1,53 +1,51 @@
-| `includeServerOutInPodLog` | Boolean | If true (the default), then the server .out file will be included in the pod's stdout. |
-| `introspectVersion` | string | If the domain is running, the operator will restart its introspector job when this value is changed. This field is ignored when the domainHomeSourceType is FromModel. See also overridesConfigurationStrategy. |
-| `logHome` | string | The in-pod name of the directory in which to store the domain, Node Manager, server logs, server  *.out, and optionally HTTP access log files if `httpAccessLogInLogHome` is true. Ignored if logHomeEnabled is false. |
-| `logHomeEnabled` | Boolean | Specified whether the log home folder is enabled. Not required. Defaults to true if domainHomeSourceType is PersistentVolume; false, otherwise. |
-| `managedServers` | array of [Managed Server](#managed-server) | Configuration for individual Managed Servers. |
-| `maxClusterConcurrentStartup` | number | The maximum number of Managed Servers that the operator will start in parallel for a cluster, if `maxConcurrentStartup` is not specified at the cluster level. A value of 0 (the default) means there is no configured limit. |
-| `replicas` | number | The number of managed servers to run in any cluster that does not specify a replica count. |
-| `restartVersion` | string | If present, every time this value is updated the operator will restart the required servers. |
-| `serverPod` | [Server Pod](#server-pod) | Configuration affecting server pods. |
-| `serverService` | [Server Service](#server-service) | Customization affecting ClusterIP Kubernetes services for WebLogic Server instances. |
-| `serverStartPolicy` | string | The strategy for deciding whether to start a server. Legal values are ADMIN_ONLY, NEVER, or IF_NEEDED. |
-| `serverStartState` | string | The state in which the server is to be started. Use ADMIN if server should start in the admin state. Defaults to RUNNING. |
-| `webLogicCredentialsSecret` | [Secret Reference](k8s1.13.5.md#secret-reference) | The name of a pre-created Kubernetes secret, in the domain's namespace, that holds the username and password needed to boot WebLogic Server under the 'username' and 'password' fields. |
+| `dataHome` | string | An optional, directory in a server's container for data storage of default and custom file stores. If dataHome is not specified or its value is either not set or empty then the data storage directories are determined from the WebLogic domain home configuration. |
+| `domainHome` | string | The directory containing the WebLogic domain configuration inside the container. Not required. Defaults to /shared/domains/domains/<domainUID> if domainHomeSourceType is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if domainHomeSourceType is Image. Defaults to /u01/domains/<domainUID> if domainHomeSourceType is FromModel. |
```

Comment:
domainHomeSourceType -> `domainHomeSourceType` (code font is a "nice to have" for code syntax when included in a description. If it's too much trouble to change them all, I understand)

Reply:
Let me see if I can do something... the Java code is the single source both for the CRD description where I probably don't want the backticks and here where I would.